.eslintrc

```diff
@@ -30,6 +30,7 @@
     }],
     "valid-jsdoc": "warn",
     "semi": ["error", "always"],
-    "quotes": ["error", "double", { "allowTemplateLiterals": true }]
+    "quotes": ["error", "double", { "allowTemplateLiterals": true }],
```

Comment:
Modifying ESlint rules is great, but really out of scope for a PR about emitters.

I would imagine 2 PRs that are separate:
- One PR the rules for quotes/any.
- One PR that changes the emitter.

Reply:
> I would imagine 2 PRs that are separate:

I think the point of this change is that the existing linter rules was causing this PR to generate warnings during the build. See https://github.com/cloudevents/sdk-javascript/pull/366#discussion_r522475661. I'm not sure of the benefit returned by extracting this small change into a separate PR, landing that first, then landing this. In fact, I recall recently that you were not happy about the number of commits in this repo - this seems counter to that complaint! :wink:

Reply:
Hey, I'd quickly approve a change/fix to the eslint rules, but I don't want to approve a change that includes `while (msg === undefined) {`. The eslint change is a different decision than emitter functionality and would like to be merged even if any emitter PRs were not merge. Does that make sense?

Reply:
> The eslint change is a different decision than emitter functionality and would like to be merged even if any emitter PRs were not merge. Does that make sense?

Sure. It just seems pedantic to demand this.

Reply:
Lance, I'm not demanding. This is just a review suggestion.

Reply:
The goal was to merge the PR in a timely manner so at that time we thought it would make sense to have only one PR

Reply:
Having a PR reviewed in a timely manner is fair. In general, it would be easier to not go back and forth and review faster with a GitHub issue and small PR. There have been discussions about removing this before, and we just talked about splitting this repo into different modules, so it's difficult to approve this PR right away.

That said, this repo is likely going to change a bit with the split, and I don't want to block you if Lance approves.